### PR TITLE
Short blogpost about new features

### PR DESCRIPTION
**Pull request by ghost**
@hpk42 is this what you had in mind? Did I miss anything? Not sure if all of this is released yet.

### CODE REVIEW

**Comment by Simon-Laux** [comment]
for desktop, please wait for 1.3.1; 1.3 has some bugs we are currently fixing.

**Comment by r10s** [comment]
@lefherz great that you push that forward, wrt account-switching: i'd like to tune that down, it is fine to mention it, but if there is only one android-screenshot, i suggest to use one from the new in-chat-search.

also ios is not yet through the apple review, btw.

File: _posts/2020-05-06-may-features.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Delta Features for May - ECC Keys, Autodelete Messages, Scan QR Codes With Desktop
+author: compl4xx
+image: ../assets/blog/screenshots/2020-05-06-delta-chat-desktop-scan-qr-code.jpg
+---
+
+**While the world was on hold, the Delta Chat development silenty continued.
+Here is a short list of features you can try out with the [new
+releases](https://get.delta.chat) coming this May!**
```
Comment (hpk42) [inline _posts/2020-05-06-may-features.md]: don't like this intro much to be honest.   Would just say "New Delta chat releases! thanks to all the early testers and all the work that went into the new releases on Android, iOS and Desktop. Here is an overview on what you can expect" or something along the lines.  There is enough references to covid19 everywhere, and it's also not really true -- covid19 happenings affected and continue to affect me quite a bit and also others i think ....

Comment (ghost) [reply _posts/2020-05-06-may-features.md]: Okay, I just took that :) wanted to make it a bit hopeful, but of course I don't know that well what you are going through.

**Comment by ghost** [comment]
I addressed your comments, feel free to merge as soon as all the releases are out :)

File: _posts/2020-05-06-may-features.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+*You can now scan QR codes with the Desktop Client.*
+
+## Android 1.6
+
+- Auto-delete messages from server, and/or from your device
+- Have more than one account and switch between them (experimental)
+- Search messages in a chat
+- Search through the in-app help
+- For new accounts, ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.
```
Comment (comradekingu) [inline _posts/2020-05-06-may-features.md]:
```suggestion
- For new accounts, Ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.
```
As per https://tools.ietf.org/html/rfc8032 and https://tools.ietf.org/html/rfc8080

File: _posts/2020-05-06-may-features.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+![Search through chats in the Android App](../assets/blog/screenshots/2020-05-06-delta-chat-android-chat-search.png)
+
+*Search through all messages in a chat, and jump between the found messages.*
+
+## iOS 1.6
+
+- Auto-delete messages from server, and/or from your device
+- Get started easier with the new welcome screen
+- Create account by scanning a QR code
+- For new accounts, ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.
```
Comment (comradekingu) [inline _posts/2020-05-06-may-features.md]:
```suggestion
- For new accounts, Ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.
```

File: _posts/2020-05-06-may-features.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+title: Delta Features for May - ECC Keys, Autodelete Messages, Scan QR Codes With Desktop
+author: compl4xx
+image: ../assets/blog/screenshots/2020-05-06-delta-chat-desktop-scan-qr-code.jpg
+---
+
+**New Delta chat releases! Thanks to all the early testers and all the work
+that went into the new releases on Android, iOS and Desktop. Here is an
+overview on what you can expect:**
```
Comment (comradekingu) [inline _posts/2020-05-06-may-features.md:9]:
```suggestion
overview of what you can expect:**
```

File: _posts/2020-05-06-may-features.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+## iOS 1.6
+
+- Auto-delete messages from server, and/or from your device
+- Get started easier with the new welcome screen
+- Create account by scanning a QR code
+- For new accounts, ed25519 keys are generated. The new keys are smaller and faster. [This saves traffic](https://delta.chat/en/2020-03-11-reduced-message-size) and time.
+
+![Delete Messages Automatically on iOS and Android.](../assets/blog/screenshots/2020-05-06-delta-chat-ios-autodelete.jpg)
+
+*You can automatically have your messages deleted to save space on your phone or protect yourself against device theft.*
```
Comment (comradekingu) [inline _posts/2020-05-06-may-features.md]:
```suggestion
*You can automatically have your messages deleted to save space on your device or protect it against theft.*
```
Identity/information theft?

File: _posts/2020-05-06-may-features.md (hunk 4 above)
Comment (comradekingu) [inline _posts/2020-05-06-may-features.md]:
```suggestion
overview of what to expect:**
```

**Comment by ghost** [comment]
Thanks @comradekingu for these sensible suggestions :) and for reminding me of this draft, I had actually forgotten it :D 

@hpk42 now that the releases are out, does anything speak against merging this?

**Comment by r10s** [comment]
ios 1.8 is still in "rejected" state, see https://github.com/deltachat/deltachat-ios/issues/669

but maybe we can say that it is "in work" or so.

**Comment by ghost** [comment]
I mean, if it's only a few days, then we could just wait a bit more. Not that it makes much of a difference now^^

**Review by r10s — APPROVED** [review]

**Comment by r10s** [comment]
ios is accepted, android+desktop quite stable and may is close to end - i think, we should merge that :)

**Pull request merged**